app/src/main/java/org/fossasia/pslab/others/ZoomLayout.java

```diff
@@ -0,0 +1,132 @@
+package org.fossasia.pslab.others;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.ScaleGestureDetector;
+import android.view.View;
+import android.widget.RelativeLayout;
+
+public class ZoomLayout extends RelativeLayout implements ScaleGestureDetector.OnScaleGestureListener {
+
+    private static final String TAG = "ZoomLayout";
```

Comment:
this may be not required

Reply:
It's required. I have used it as a tag while printing Log statements